### PR TITLE
Clear target major

### PR DESCRIPTION
**Pull request by jonmv**
@mpolden please review. Let's merge after next release, and then revert after this is also out.

### CODE REVIEW

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java [hunk 1]
```diff
@@ -273,6 +273,10 @@ public void writeVersionStatus(VersionStatus status) {
         curator.set(versionStatusPath(), asJson(versionStatusSerializer.toSlime(status)));
     }
 
+    public void clearTargetMajor() {
+        curator.delete(root.append("ugprader").append("targetMajorVersion"));
```
Comment (mpolden) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java:277]: Typo

Comment (jonmv) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java:277]:
```suggestion
        curator.delete(root.append("upgrader").append("targetMajorVersion"));
```

(((＼（✘෴✘）／)))

Comment (jonmv) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java]: ┏( ﾟ益ﾟ)ゞ

**Comment by jonmv** [comment]
Thanks!

**Review by mpolden — APPROVED** [review]

**Pull request merged**